gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```

I suggest bumping the version to `6.9`. For now I'd like to postpone upgrading to version 7.x until we can ensure compatibility with newer Android Gradle Plugin versions

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
```